Mathlib/Topology/Instances/ENNReal.lean

```diff
@@ -1083,6 +1083,13 @@ theorem finset_card_const_le_le_of_tsum_le {ι : Type*} {a : ι → ℝ≥0∞}
     _ ≤ c := tsum_le_c
 #align ennreal.finset_card_const_le_le_of_tsum_le ENNReal.finset_card_const_le_le_of_tsum_le
 
+theorem tsum_fiberwise (f : β → ℝ≥0∞) (g : β → γ) :
+    ∑' (x : γ), ∑' (b : (g ⁻¹' {x})), f b = ∑' i : β, f i := by
+  apply HasSum.tsum_eq
+  let equiv := Equiv.sigmaFiberEquiv g
+  apply ((equiv.hasSum_iff).mpr (Summable.hasSum ENNReal.summable)).sigma
+  apply (fun _ ↦ ((ENNReal.summable).hasSum_iff).mpr rfl)
```

Comment:
```suggestion
    ∑' x, ∑' b : g ⁻¹' {x}, f b = ∑' i, f i := by
  apply HasSum.tsum_eq
  let equiv := Equiv.sigmaFiberEquiv g
  apply (equiv.hasSum_iff.mpr ENNReal.summable.hasSum).sigma
  exact fun _ ↦ ENNReal.summable.hasSum_iff.mpr rfl
```
Thanks a lot, this looks great! I have removed a few useless parentheses (this tends to increase readability) and type annotations, and changed the last `apply` to `exact` (because this indicates this is the end of the proof). I have also used dot notation. This is just to show you some tricks, your proof was already perfectly acceptable!